setup.py

```diff
@@ -30,7 +30,7 @@
 with open('README.md') as f:
     description_text = f.read()
 
-install_req = ["lxml", "pyyaml==4.2b4", "rdflib", "docopt", "pathlib"]
+install_req = ["lxml", "pyyaml", "rdflib", "docopt", "pathlib"]
```

Comment:
Do older versions of pyyaml exhibit the issue?  Do we need to specify that it should be `>4.2` (or `>=4.3`) to avoid using older versions?

Reply:
There is no official 4 release. Maybe we should use 5.1 as a new base line.